### ISSUE TITLE
Training the model

### ISSUE THREAD

**Issue by Yahya-khodr** [yl4579/StyleTTS#58]
Hello,
I'm trying to train the libritts model on a new dataset,
I did create new folder for the audio files and a train_data.txt file with the txt and the audio just like the desired format mentioned,
But I what about the the val_data and train_data in the config.yaml file ?
`train_data: "Data/train_list_libritts.txt"
val_data: "Data/val_list_libritts.txt"
`
Should I update my new .txt file with the val_data or train_data ? 
Thank you

**Comment by yl4579** [comment]
Yes, it is just list of data, so it should correspond to the list of samples of your new dataset.
